src/helpers/WalletLanguage.ts

```diff
@@ -0,0 +1,19 @@
+import * as english from './languages/english.json';
+import * as spanish from './languages/spanish.json';
+
+export default class WalletLanguage {
+    getLanguage() {
+        const lang: any = ((navigator.languages && navigator.languages[0] && navigator.language) || 'en').substr(0, 2);
```

Comment:
why `any`? Looks like this would always be a string? The function is missing the return type.